pkg/apis/openstackproviderconfig/v1alpha1/types.go

```diff
@@ -81,6 +80,37 @@ type OpenstackProviderSpec struct {
 	ConfigDrive *bool `json:"configDrive,omitempty"`
 }
 
+type Tags struct {
+	// Tags added to all resources created by cluster
+	// Default tags include: cluster-api-provider-openstack, clusterName
```

Comment:
I think cluster-api-provider-openstack:clusterName should be enough?

Reply:
These are the default tags that the project already had. I would prefer not to change them in this pull request, but can in another one.